pkg/scheduler/partition.go

```diff
@@ -869,44 +869,54 @@ func (pc *PartitionContext) allocate(result *objects.AllocationResult) *objects.
 	// find the node make sure it still exists
 	// if the node was passed in use that ID instead of the one from the allocation
 	// the node ID is set when a reservation is allocated on a non-reserved node
-	var nodeID string
 	alloc := result.Request
+	targetNodeID := result.NodeID
+	var srcNodeID string
```

Comment:
Rename to `reservedNodeID` is more logical than `srcNodeID`
The reserved node is only used if the result is `Unreserved` or `AllocatedReserved` This retrieval and the logging should be moved inside that check (line 911) and not clutter the rest of the code.